src/index.ts

```diff
@@ -16,6 +20,22 @@ router.get("/", async () => {
   });
 });
 
+router.put("/roles/create", createGuildRoleHandler);
+
+router.put("/roles/add", addGroupRoleHandler);
+
+router.post("/", async (request, env) => {
+  const message: discordMessageRequest = await request.json();
+  if (message.type === InteractionType.PING) {
+    return new JSONResponse({
+      type: InteractionResponseType.PONG,
+    });
+  }
+  if (message.type === InteractionType.APPLICATION_COMMAND) {
+    return baseHandler(message, env);
+  }
+  return new JSONResponse(response.UNKNOWN_INTERACTION, { status: 400 });
+});
```

Comment:
the same is present below why are we writing it again?

I checked out your PR locally and checkout line 27 and line 41

Please fix it

![image](https://github.com/Real-Dev-Squad/discord-slash-commands/assets/57758447/f96f0c86-7fdc-48f3-ad86-d4b2189ea7b3)